src/segment-loader.js

```diff
@@ -717,6 +717,43 @@ export default class SegmentLoader extends videojs.EventTarget {
     };
   }
 
+  abortRequestEarly_(stats) {
+    const firstByteReceived = Date.now() - stats.firstByteReceived;
```

Comment:
well here `const firstByteReceived` is the time that has passed since we received the first byte of data. `msSinceFirstByteReceived` maybe